src/components/assetlib_projects/asset_library_projects.tscn

```diff
@@ -0,0 +1,223 @@
+popup/item_1/text = "Templates"
+popup/item_1/id = 8
+popup/item_2/text = "Projects"
+popup/item_2/id = 9
+popup/item_3/text = "Demos"
+popup/item_3/id = 10
+
+[node name="VSeparator2" type="VSeparator" parent="FilterSortOptions"]
+layout_mode = 2
+
+[node name="Label3" type="Label" parent="FilterSortOptions"]
+layout_mode = 2
+text = "Site:"
+
+[node name="SiteOption" type="OptionButton" parent="FilterSortOptions"]
+unique_name_in_owner = true
+editor_description = "Also only has one option in the official project manager.  I don't know why it's there, since there's seemingly no way to add other sources for assets."
+layout_mode = 2
+size_flags_horizontal = 3
+item_count = 1
+selected = 0
+popup/item_0/text = "godotengine.org (Official)"
+popup/item_0/id = 0
+
+[node name="VSeparator3" type="VSeparator" parent="FilterSortOptions"]
+layout_mode = 2
+
+[node name="SupportOptions" type="MenuButton" parent="FilterSortOptions"]
+unique_name_in_owner = true
+layout_mode = 2
+text = "Support"
+item_count = 3
+popup/item_0/text = "Official"
+popup/item_0/checkable = 1
+popup/item_0/checked = true
+popup/item_0/id = 0
+popup/item_1/text = "Community"
+popup/item_1/checkable = 1
+popup/item_1/checked = true
+popup/item_1/id = 1
+popup/item_2/text = "Testing"
+popup/item_2/checkable = 1
+popup/item_2/id = 2
+
+[node name="MarginContainer" type="MarginContainer" parent="."]
+layout_mode = 2
+size_flags_vertical = 3
+
+[node name="ScrollContainer" type="ScrollContainer" parent="MarginContainer"]
+layout_mode = 2
+size_flags_vertical = 3
+
+[node name="VBoxContainer" type="VBoxContainer" parent="MarginContainer/ScrollContainer"]
+layout_mode = 2
+size_flags_horizontal = 3
+
+[node name="NavigationButtons" type="HBoxContainer" parent="MarginContainer/ScrollContainer/VBoxContainer"]
+unique_name_in_owner = true
+visible = false
+layout_mode = 2
+alignment = 1
+
+[node name="FirstButton" type="Button" parent="MarginContainer/ScrollContainer/VBoxContainer/NavigationButtons"]
+layout_mode = 2
+text = "First"
+
+[node name="PreviousButton" type="Button" parent="MarginContainer/ScrollContainer/VBoxContainer/NavigationButtons"]
+layout_mode = 2
+text = "Previous"
+
+[node name="VSeparator" type="VSeparator" parent="MarginContainer/ScrollContainer/VBoxContainer/NavigationButtons"]
+layout_mode = 2
+
+[node name="PageButtons" type="HBoxContainer" parent="MarginContainer/ScrollContainer/VBoxContainer/NavigationButtons"]
+layout_mode = 2
+
+[node name="VSeparator2" type="VSeparator" parent="MarginContainer/ScrollContainer/VBoxContainer/NavigationButtons"]
+layout_mode = 2
+
+[node name="NextButton" type="Button" parent="MarginContainer/ScrollContainer/VBoxContainer/NavigationButtons"]
+layout_mode = 2
+text = "Next"
+
+[node name="LastButton" type="Button" parent="MarginContainer/ScrollContainer/VBoxContainer/NavigationButtons"]
+layout_mode = 2
+text = "Last"
+
+[node name="AssetList" type="HFlowContainer" parent="MarginContainer/ScrollContainer/VBoxContainer"]
```

Comment:
Replace to GridContainer since the Flow one is not able to be resized well (i want the items to be strictly in a grid)

In order to calculate how much columns should be in GridContainer depending on the current asset_library_projects.gd size use this:
```gdscript
func _update_columns():
	var new_columns = size.x / (450 * Config.EDSCALE)
	new_columns = max(1, new_columns)
	if new_columns != _asset_list.columns:
		_asset_list.columns = new_columns
```
the function should be called every time the _asset_list content is changed and the container is resized. 

btw the same logic is used in Godot's Project Manager, so it is possible to take a look how the feature was implemented there.